core/Azure.Mcp.Core/src/Areas/Server/Resources/azure-rules.txt

```diff
@@ -12,9 +12,10 @@ sure the user is discussing Azure Functions; do not call it otherwise.
 - Use Azure Functions Deployment Best Practices: When deploying Azure Functions apps to Azure or preparing for 
 the deployment, invoke your `bestpractices` tool if available. Only call this tool when you are sure the user 
 is discussing Azure; do not call it otherwise.
-- Use Azure AI Code Generation Best Practices: When generating code for AI applications, AI agents, workflows, 
-or chatbots in Azure, invoke your `azureaibestpractices` tool if available. Only call this tool when you are sure 
-the user is discussing Azure and code generation; do not call it for Azure resource operations.
+- Use Azure AI App Code Generation Best Practices: When generating code for AI applications, AI agents, workflows, 
+or chatbots, invoke your `get_bestpractices_ai_app` command (under `get_bestpractices` tool) if tool available
+Call this tool first before creating any plans, todos or code.
```

Comment:
Should this be a more generic rule for all Azure best practices scoped to Azure related intents - @fanyang-mono